### PR TITLE
QT - Improve accessibility of stream dialog

### PR DESCRIPTION
**Pull request by Oreonan**

### CODE REVIEW

File: Client/qtTeamTalk/streammediafile.ui [hunk 1]
```diff
@@ -42,7 +45,7 @@
            <string>Browse</string>
           </property>
           <property name="text">
-           <string>...</string>
+           <string>... &amp;B</string>
```
Comment (bear101) [inline Client/qtTeamTalk/streammediafile.ui:48]: Should we instead change this to an icon as well (same as Refresh button)? It looks a bit weird with "...B"

**Comment by Oreonan** [comment]
If you think that it's can be better, I don't see any inconvenient.

**Comment by bear101** [comment]
But it's easy enough to know that "R" is a shortcut to "Refresh" although it's an icon?

**Comment by Oreonan** [comment]
Currently, for "refresh" button, screenreader says: "Refresh button 
alt+R", so we know what shortcut is for what button :)

**Comment by bear101** [comment]
Ok, then I change it to icon.

--
Kind regards

Bjørn Damstedt Rasmussen
www.bearware.dk

**Comment by bear101** [comment]
Do we need the three dots for "R"? I.e. "...&R"

**Comment by Oreonan** [comment]
As you can removed this for B, it's more logic to remove also for R.

**Pull request merged**